node/ChainwebNode.hs

```diff
@@ -496,7 +496,7 @@ pkgInfoScopes =
 -- KILLSWITCH for version 1.4
 --
 killSwitchDate :: Maybe String
-killSwitchDate = Just "2020-01-15T00:00:00Z"
+killSwitchDate = Just "2020-02-15T00:00:00Z"
```

Comment:
February 15 is a Saturday.  Should we move this to a Wednesday like we've been doing?  @slpopejoy

Reply:
How about Feb 13 or Feb 20?